R/aaa-auto.R

```diff
@@ -330,6 +330,19 @@ simple_interconnected_islands_game_impl <- function(islands.n, islands.size, isl
   res
 }
 
+chung_lu_game_impl <- function(expected.out.deg, expected.in.deg=NULL, loops=TRUE, variant=ORIGINAL) {
```

Comment:
There needs to be translation code for `igraph_chung_lu_t` values.

Reply:
Does it make sense to make it in seperate PR?

Reply:
I think they're might be warnings if `=ORIGINAL` is left in there. I suggest you disable auto generation for this function at the moment, then we can handle it in a later PR